module/executiondatasync/tracker/storage.go

```diff
@@ -52,13 +52,13 @@ const blobRecordKeyLength = 1 + 8 + blobs.CidLength
 func makeBlobRecordKey(blockHeight uint64, c cid.Cid) []byte {
 	blobRecordKey := make([]byte, blobRecordKeyLength)
 	blobRecordKey[0] = prefixBlobRecord
-	binary.LittleEndian.PutUint64(blobRecordKey[1:], blockHeight)
+	binary.BigEndian.PutUint64(blobRecordKey[1:], blockHeight)
```

Comment:
Details in #6113

Reply:
thanks. can you update the other cases we used `LittleEndian` to use `BigEndian` as well. It's not critical but better to stay consistent

Reply:
I would expect this to have been caught by tests. Can you figure out why it wasn't and improve the tests? It sounds like the tracker would effectively not work at all.

Reply:
[fa5fc2e](https://github.com/onflow/flow-go/pull/6109/commits/fa5fc2ef02cb1f002a1697d8540c996f76b08dc3)